movingpandas/trajectory_cleaner.py

```diff
@@ -148,4 +150,9 @@ def _clean_traj(self, traj, v_max=None, units=UNITS(), alpha=3):
             prev = curr
 
         out_traj.df.drop(ixs, inplace=True)
+
+        if not out_traj.is_valid():
+            warnings.warn(f"Cannot clean trajectory {out_traj.id}.", UserWarning)
```

Comment:
Lets explain to the user what's going on, e.g.

"Cleaning failed for trajectory {out_traj.id} due to invalid result (e.g. <2 points remaining). Returning original trajectory."